### ISSUE TITLE
Implement CrowdActions View

### ISSUE THREAD

**Issue by dromerolovo** [CollActionteam/collaction_cms#37]
Check Figma **CrowdAction View** in Final screens for reference.

- [x] Create the UI of CrowdAction View / Text composition components
- [x] Wrap the title of crowdActions items with `GestureDetetor` or a similar widget so that clicking it triggers a navigation to crowdAction view.
- [x] Configure `GoRouter` to support this. Would be a dynamic routing.
- [x] Create pertinent Domain entities and values like commitments.
- [x] Create a bloc that deals with "getting" the crowdAction, lets call it: `SelectedCrowdActionBloc` or something like that. 
Basically when an admin clicks a crowdAction's title on the table. it will trigger a NavigationEvent and also trigger a bloc event so that the selected crowdAction is "uploaded into a `SelectedCrowdActionBloc`". And the widget that manages the presentation can consume the state.
